README.md

```diff
@@ -90,8 +97,12 @@ lcmap-cli requires an config file at ~/.usgs/lcmap-cli.edn.
                   :pixel "/pixel"
                   :segment "/segment"
                   :annual-prediction "/annual_prediction"
+                  :products "/products"
+                  :maps "/maps"
                   :segment-instance-count 1
-                  :segment-sleep-for 1000}
+                  :request-instance-count 1
+                  :segment-sleep-for 1000
+                  :product-doy "07-01"}
```

Comment:
doy should be an integer...  

In this case does 07-01 == mm-yy or mm-dd or dd-mm or yy-mm?

Reply:
I don't think that'd work.  If Products are supposed to be generated for July 1, that integer value is going to be different every 4 years due to leap years.   "07-01" in this case is mm-dd